src/com/elixr/training/Main.java

```diff
@@ -3,44 +3,60 @@
 import java.io.File;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.StringTokenizer;
 import java.lang.String;
+import java.sql.*;
+
+import com.elixrdb.training.JdbcConnection;
 
 public class Main {
-    static  final String TXTFILE = ".txt" ;
-    static final String JSONFILE = ".json";
-    public static void main(String[] args) {
+    static JdbcConnection database = new JdbcConnection();
+    static int count = 0;
+    static String inputFilePath;
+    static String searchWord;
 
+    public static void main(String[] args) {
         if (args.length != 2) {
-            System.out.println(" File path and word to search are missing. Exiting the program... ");
+            System.out.println(Constants.ERROR_MESSAGE_ARGUMENTS_NOT_FOUND);
+            try {
+                database.dbOperation(inputFilePath, searchWord, Constants.STATUS_FAILURE, count, Constants.ERROR_MESSAGE);
```

Comment:
The last parameter is the actual error message, right? 
But Constants.ERROR_MESSAGE is just "0", so all database entries will be just 0
You should save the actual message you are printing. The message you are printing in sout

Update this in all other places

Reply:
I have updated to actual message